TTBot/DataAccess/Events.cs

```diff
@@ -56,5 +56,13 @@ public async Task<EventsWithCount> GetEventByMessageIdAsync(ulong messageId)
                 return await con.SingleAsync<EventsWithCount>(e => e.MessageId == messageId.ToString());
             }
         }
+
+        public async Task<EventsWithCount> GetEventByShortname(string shortname)
+        {
+            using (var con = await _conFactory.OpenAsync())
+            {
+                return (await con.SelectAsync<EventsWithCount>(ev => (ev.ShortName.ToLower() == shortname.ToLower()))).SingleOrDefault();
```

Comment:
This can throw an error as event shortnames are only unique per channel currently. I'd suggest making a change to make event short names unique per Guild and also filter on the Context.Guild.Id in this.